modules/metrics/src/main/java/org/apache/ignite/internal/metrics/DistributionMetric.java

```diff
@@ -59,18 +60,34 @@ public class DistributionMetric extends AbstractMetric implements CompositeMetri
     /** Bounds of measurements. */
     private final long[] bounds;
 
+    /** Value string formatter or {@code null} to use default formatter. */
```

Comment:
It seems to me, that this change should be applied to all metric classes: `IntMetric`, `IntGaguge` etc

Reply:
Maybe we will do it when we start to use them.
Currently, it just adds code to review here.

Reply:
Honestly, I don't see any difference between `IngGaguge` and `LongGaguge`, and therefore this improvement should be added to both, I think. Please create a ticket to address this.

Reply:
I removed all the string formatters in metriics.
We will add them in the future after discussion.